pkg/sql/sql.go

```diff
@@ -123,6 +124,17 @@ func (c *client) CreateReadPermission(r clouddriver.ReadPermission) error {
 	return db.Error
 }
 
+// A Kubernetes cluster is of kind deployment, statefulSet, replicaSet, ingress, service, and daemonSet.
+func (c *client) ListKubernetesClustersByApplication(spinnakerApp string) ([]kubernetes.Resource, error) {
+	var rs []kubernetes.Resource
+	db := c.db.Select("account_name, cluster").
+		Where("spinnaker_app = ? AND kind in (deployment, statefulSet, replicaSet, ingress, service, daemonSet)",
```

Comment:
Do these need to be enclosed in quotes?
```
kind in ('deployment', 'statefulSet', 'replicaSet', 'ingress', 'service', 'daemonSet')
```

Reply:
They do! I fixed here: https://github.com/billiford/go-clouddriver/pull/13/commits/5927267594ebaf7f5efbd639a4c41a48691594c8.